text/0000-embedded-refract.md

```diff
@@ -0,0 +1,361 @@
+    }
+  },
+  "first_name": "John",
+  "last_name": "Doe",
+  "age": 28,
+  "scores": [150, 202, 145]
+}
+```
+
+This is what this would like in full Refract.
+
+```json
+{
+  "element": "object",
+  "meta": {
+    "id": "bowler-103"
+  },
+  "content": [
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "first_name"
+        },
+        "value": {
+          "element": "string",
+          "content": "John"
+        }
+      }
+    },
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "first_name"
+        },
+        "value": {
+          "element": "string",
+          "content": "Doe"
+        }
+      }
+    },
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "age"
+        },
+        "value": {
+          "element": "number",
+          "content": 28
+        }
+      }
+    },
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "scores"
+        },
+        "value": {
+          "element": "array",
+          "content": [
+            {
+              "element": "number",
+              "content": 150
+            },
+            {
+              "element": "number",
+              "content": 202
+            },
+            {
+              "element": "number",
+              "content": 145
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+```
+
+## Refracting Value Elements
+
+If values do not have any meta values or attributes, there is no need to represent them as refracted. If I were to add some annotation to one of the values, it would show up as Refract and would be defined as Refracted by use of the `refract` property keyword.
+
+```json
+{
+  "first_name": "John",
+  "last_name": "Doe",
+  "age": {
+    "refract": {
+      "element": "number",
+      "meta": {
+        "id": "bowner-103-age"
+      },
+      "content": 28
+    }
+  },
+  "scores": [150, 202, 145]
+}
+```
+
+Above, I've added an ID to the age of the bowler. Notice how I just refracted that value in that specific place. This is also the same as:
+
+```json
+{
+  "refract": {
+    "element": "object",
+    "meta": {
+      "id": "bowler-103"
+    },
+    "content": {
+      "age": {
+        "element": "number",
+        "meta": {
+          "id": "bowner-103-age"
+        },
+        "content": 28
+      }
+    }
+  },
+  "first_name": "John",
+  "last_name": "Doe",
+  "scores": [150, 202, 145]
+}
+```
+
+This will also support full member elements, so even this below is equivalent (note that it's a slight modification from the full Refract):
+
+```json
+{
+  "refract": {
+    "element": "object",
+    "meta": {
+      "id": "bowler-103"
+    },
+    "content": [
+      {
+        "element": "member",
+        "content": {
+          "key": "age",
```

Comment:
I don't have a concrete example, but this is raising a red flag for me since it may lead to ambiguous cases where we cannot determine if a value is refracted. The first element of the first `content` array has no `refract` key and so an implementer must determine whether it is refracted by the `element` and `content` keys instead. I'm worried about this.